pgoapi/pgoapi.py

```diff
@@ -178,6 +183,14 @@ def __init__(self, config):
         self.should_catch_pokemon = config.get("CAPTURE", {}).get("CATCH_POKEMON", True)
         self.max_catch_attempts = config.get("CAPTURE", {}).get("MAX_CATCH_ATTEMPTS", 10)
 
+        self.new_forts = []
+        self.cache_filename = './cache/cache ' + str(config["location"]) + str(self.STAY_WITHIN_PROXIMITY)
```

Comment:
i think `config["location"]` could lead to some problems here:
- First you should probably use `config.get("location", "unknown")` or something
- Next there can be issues file prohibited characters for filenames on different OSs, maybe use a hash on the location or something like that?